src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to ingest new metrics in the Observability Platform
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
```

Comment:
I thought you wanted to take the flat structure now? Looking at it I feel that this 3 levels down are a bit too many clicks to get there without any added value. Wdyt?

Reply:
That's what I originally thought but then I found that that worked quite well with the separation but maybe this does not need 3 layers?

I was definetly thinking that later on we would have something like:

observavbility
  data-ingestion
  data-exploration
    creating dashboards
    exploring metrics and logs
  exporting-data

So i added the 3 layers now but i can change this

Reply:
Yeah, guess it's fine that way. Let's keep it for now